### PR TITLE
It1 from csv 

### PR DESCRIPTION
**Pull request by aleish-m**
Added functionality to from_csv method to read CSV files into the class file and test they were a match to the original data file.

### CODE REVIEW

**Review by stephenfabian — DISMISSED** [review]
Glad we found a way to read the CVS.  Viewing the CSVs through the table method was far easier to manage than our first attempt.

**Review by knestler — DISMISSED** [review]
Good place to start methods on.

**Review by KevinT001 — DISMISSED** [review]

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
+  end
 
     def self.from_csv(locations)
-      StatTracker.new
+      games = CSV.table(locations[:games], headers: true, header_converters: :symbol)
+      teams = CSV.table(locations[:teams], headers: true, header_converters: :symbol)
+      game_teams = CSV.table(locations[:game_teams], headers: true, header_converters: :symbol)
 
+      StatTracker.new(games, teams, game_teams)
+      # require "pry"; binding.pry
```
Comment (aleish-m) [inline lib/stat_tracker.rb]: I need to remove this pry

Comment (KevinT001) [reply lib/stat_tracker.rb]: Good catch

Comment (aleish-m) [reply lib/stat_tracker.rb]: fixed

**Review by knestler — APPROVED** [review]

**Review by stephenfabian — APPROVED** [review]
Surprised that those arguments didn't do anything.  Thanks for removing those.

**Review by KevinT001 — APPROVED** [review]
Thank you for explaination on csv.table and how it can be used to shorten things

**Pull request merged**